docs/cli.md

```diff
@@ -197,6 +197,56 @@ Usage:
       --validate=false: If true, use a schema to validate the input before sending it
 ```
 
+#### resize
+Set a new size for a resizable resource (currently only Replication Controllers)
+
+Resize also allows users to specify one or more _preconditions_ for the resize action.  The two currently supported options are current size (```--current-size```) or resource version (```--resource-version```).  If a precondition is specified, it is validated before the resize is attempted, and it is guaranteed that the precondition holds true when the resize is sent to the server.
+
+Examples:
+```sh
+  $ kubectl resize replicationcontrollers foo 3
+  resized
+```
+
+```sh
+  # will only execute if the current size is 3
+  $ kubectl resize --current-size=2 replicationcontrollers foo 3
+```
+
+Usage:
+```
+  kubectl resize [--resource-version] [--current-size] <resource> <id> <count> [flags]
```

Comment:
`--resource-version=<version> --current-size=<size>`

We plan to make several commands (e.g., get and delete are obvious ones) accept multiple resource names on the command line. It's not done yet. 

Why don't you want to make count a flag? With a short flag (e.g., -n for --new-size), it wouldn't be significantly longer, and would remove the unique, order-dependent interpretation of that command-line argument.